site/source/tutorials/005_form_widgets/demo/finished/biz-e-corp/src/styles/workerForm.css

```diff
@@ -0,0 +1,29 @@
+.workerForm {
+	margin-bottom: 40px;
+	text-align: center;
+}
+
+.workerForm fieldset,
+.workerForm label {
+	display: inline-block;
+	text-align: left;
+}
+
+.workerForm label {
+	margin-right: 10px;
+}
+
+.nameField {
+	border: 0;
+	margin: 0;
+	padding: 0;
+}
+
+.nameLabel {
+	font: 14px/1 sans-serif;
```

Comment:
An actual app would probably override the `label` styles, but I thought it might not be a good fit for the tutorial